### PR TITLE
Remove `A1` assertion from worker's code

### PR DESCRIPTION
**Pull request by kziemianek**
Removes `A1` assertion from worker's code.

It's not needed since we converted it to Dynamic Assertion.

### CODE REVIEW

**Comment by Kailai-Wang** [comment]
Would you keep it for a while? At least as long as IDH hasn't switched to DA in the production release 🤔

File: tee-worker/litentry/core/stf-task/receiver/src/handler/assertion.rs [hunk 1]
```diff
@@ -276,11 +269,17 @@ where
...
 
-		Assertion::Dynamic(smart_contract_id) => lc_assertion_build::dynamic::build(
-			req,
-			smart_contract_id,
-			context.assertion_repository.clone(),
-		),
+		Assertion::Dynamic(smart_contract_id) => {
+			#[cfg(test)]
+			{
+				std::thread::sleep(core::time::Duration::from_secs(5));
```
Comment (Traf333) [inline tee-worker/litentry/core/stf-task/receiver/src/handler/assertion.rs:275]: for what reason do we need the sleep? Is it awaits something to be created before?

**Review by Traf333 — COMMENTED** [review]
Looks good to me :+1:

**Review by Traf333 — APPROVED** [review]

**Review by BillyWooo — CHANGES_REQUESTED** [review]
Thanks. 
Nice movement to trigger it out. But let me confirm with IDHub, whether they are ready. Then we merge this PR. I will put this PR on hold.

**Comment by jonalvarezz** [comment]
> Would you keep it for a while? At least as long as IDH hasn't switched to DA in the production release 🤔

I'd suggest the same.

is DA ready to use on production? we migrated A1 but it is not used in IDH on production.

File: tee-worker/litentry/core/stf-task/receiver/src/handler/assertion.rs (hunk 1 above)
Comment (kziemianek) [reply tee-worker/litentry/core/stf-task/receiver/src/handler/assertion.rs:275]: It's used by test to detect if two assertions are processed in parralel.

**Comment by kziemianek** [comment]
i'll reopen once we decide we need it

**Pull request closed without merging**